slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -38,12 +39,29 @@ using viam::service::slam::v1::SLAMService;
 namespace viam {
 
 static const int checkForShutdownIntervalMicroseconds = 1e5;
+
+// rgb value representing no input from a Cario pained map
+static const unsigned char defaultCairosEmptyPaintedSlice = 102;
+static const int jpegQuality = 50;
```

Comment:
What does `jpegQuality` mean?

Reply:
I tried asking this question when initially prompted about making this a variable and never got an answer.